client/src/App.scss

```diff
@@ -0,0 +1,7 @@
+.main-container {
+    background-color: rgb(226, 226, 226);
+    display: flex;
+    align-items: baseline;
+    justify-content: center;
+    min-height: calc(100vh - 4em);
```

Comment:
4em can be css variable

Reply:
but that calc is ok or there is better way to get the rest of the screen?